doc/man/man3/intro.3monocypher

```diff
@@ -195,7 +206,13 @@ vulnerability and be compromised.
 To mitigate this problem, some protocols guarantee that past messages
 are not compromised even if the long term keys are.
 This is done by generating temporary keys, then encrypting messages
-with them.
+using them.
+Monocypher provides the
+.Xr crypto_kex_xk1_init_client 3monocypher
+and
+.Xr crypto_kex_x_init_client 3monocypher
```

Comment:
This one only has limited forward secrecy. I gathered that in standard wording, you don't even advertise forward secrecy in this case. Maybe we should only mention the XK1 family?

Tough one, I also want to advertise the offline alternative. Anyway, good enough for now.